data/styles/vehicle/include/Icons.mapcss

```diff
@@ -886,27 +886,45 @@ area|z16-[amenity=water_point],
 /* 7.1 Parking */
 
 node|z15-[amenity=parking],
-area|z15-[amenity=parking]
-{icon-image: parking-m.svg; icon-min-distance: 3;}
-
+area|z15-[amenity=parking],
+{icon-image: parking-m.svg;icon-min-distance: 10;z-index: 20000;font-size: 12.5;}
```

Comment:
please remove z-index, no more z-indexes in mapcss files! :)
all priorities were moved into prio.txt files

Reply:
@pastk can kothic fail in such cases or warn at least?

Reply:
it actually just ignores `z-index:` now, so my nit was just about keeping the file clean